src/kibana/components/visualize/visualize.html

```diff
@@ -1,4 +1,4 @@
-<div ng-if="esResp.hits.total === 0"
+<div ng-if="esResp.hits.total === 0 && vis.type.name !== 'markdown'"
```

Comment:
I feel like we should figure out a better way to handle this condition. Over time we will have many "vis" objects that aren't actually backed by queries.